--- a/config.def.h	2018-05-31 01:37:02.331568929 +0700
+++ b/config.def.h	2018-05-31 01:38:00.288590335 +0700
@@ -60,5 +60,8 @@
  */
 static const struct arg args[] = {
 	/* function format          argument */
-	{ datetime, "%s",           "%F %T" },
+	{ run_command, "Volume %s ", "amixer get Master | tail -n1 | awk '{ print $5 }' | tr -d []" },
+	{ battery_state, "Battery %s ", "BAT0" },
+	{ battery_perc, "%3s%% ",    "BAT0" },
+	{ datetime, "%s",           "%F %R" },
 };
